frontend/src/components/sidebar/chanel/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'common/types/types';
+import { SubscribeChanel } from 'components/common/sidebar/sidebar-container';
+
+import styles from '../../common/sidebar/sidebar.module.scss';
```

Comment:
Not sure I get an idea. What's the reason to import styles from different component?